.circleci/config.yml

```diff
@@ -100,14 +104,15 @@ jobs:
           name: Wait for node app to start
           command: sleep 5
       - run:
-          name: integration tests
-          command: npm run int-test
+          name: Integration tests
+          command: circleci tests glob "**/*.cy.ts" | circleci tests run --command="xargs npx cypress run --reporter cypress-circleci-reporter --spec" --verbose --split-by=timings
```

Comment:
How come we're using this special command now instead of `npm run int-test`?

Reply:
This is for CircleCI to split the tests up across its different instances. DIdn't want to include this in the package.json as its specific to CircleCI

Reply:
Ah right and there's no way to encode this into the config instead and still run this normally via `npm run int-test`? If not np

Reply:
Not that I'm aware of, I believe this code needs to run within the context of the pipeline and we let circleCI do the splitting